cmd/README.md

```diff
@@ -0,0 +1,655 @@
+# Precompile Generation Tutorial
+A stateful precompile allows us to add more functionality and customization to the EVM. A stateful precompile builds on a precompile in that it adds state access. This means our precompile can manage state as well as interacting with EVM state. 
```

Comment:
Can we improve the phrasing here so we don't start two consecutive sentences in the same way?